pyspark/bigdl/keras/converter.py

```diff
@@ -816,6 +816,31 @@ def create_cropping1d(self):
         cropping = tuple(self.klayer.cropping)
         return BLayer.SpatialZeroPadding(0, 0, -cropping[0], -cropping[1])
 
+    def create_cropping2d(self):
+        if "dim_ordering" not in self.config:
+            warnings.warn("Cannot find dim_ordering from json definition. Using the default instead.")
+
+        bigdl_order = self.get_bdim_order()
+        blayer = BLayer.Cropping2D(heightCrop=self.klayer.cropping[0],
+                                   widthCrop=self.klayer.cropping[1],
+                                   data_format=bigdl_order)
+        return blayer
+
+    def create_cropping3d(self):
+        if "dim_ordering" not in self.config:
+            warnings.warn("Cannot find dim_ordering from json definition. Using the default instead.")
+        if self.klayer.dim_ordering == "th":
```

Comment:
we should use a existing mapping method  for this.

Reply:
added a 3d mapping